build.sc

```diff
@@ -19,15 +19,20 @@ val scala213 = "2.13.10"
 
 val scalaVersions = List(scala211, scala212, scala213)
 val scalaJSVersions = scalaVersions.map((_, "1.11.0"))
-val scalaNativeVersions = scalaVersions.map((_, "0.4.7"))
 
-val configName = "config"
+object Platforms {
+  val jvm = "jvm"
+  val js = "js"
+}
 
-trait CommonPublish extends CiReleaseModule with Mima {
+// NOTE: Once we publish configTestUtil we can move this down and mix it in with CommonPublish
+trait MimaCheck extends Mima {
+  override def mimaPreviousVersions = Seq("1.5.4")
```

Comment:
Do we have to update it manually after each release? Can it be automated?

Reply:
I'll dig in a bit more, but the other Mill projects I've looked at all had it explicit. And by default this is an empty Seq so I believe you need to manually do it.